test/stache-test.js

```diff
@@ -3636,7 +3636,9 @@ function makeTest(name, doc, mutation) {
 				message = 'can-stache/src/expression.js: Unable to find helper "helpme".';
 
 			canDev.warn = function (text) {
-				equal(text, message, 'Got expected message logged.');
+				if(message === text){
```

Comment:
You should add an `expect(1)` so that this test will fail if the warning is never given.